src/zzz_od/gui/view/devtools/devtools_screenshot_helper_interface.py

```
@@ -50,7 +50,7 @@ def get_widget_at_top(self) -> QWidget:
         top_widget.add_widget(self.key_save_opt)
 
         self.dodge_detect_opt = SwitchSettingCard(icon=FluentIcon.GAME, title='闪避检测',
-                                                  content='脚本识别黄光红光时，自动截图，用于捕捉误判')
+                                                  content='脚本识别黄光红光时，自动截图，用于捕捉误判；开启后自动战斗也会同步保存闪光截图')
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)